### ISSUE TITLE
Disabling Post-processing effects on cameras via config file disables camera preview

### ISSUE THREAD

**Issue by mikeskydev** [icosa-gallery/open-brush#45]
Steps to reproduce:
- Edit your Open Brush.cfg config file to disable `PostEffectsOnCapture`. Example:
```json
{
  "User": {
  },
  "Video": {
  },
  "Flags": {
    "PostEffectsOnCapture" : false,
  },
  "Export": {
  },
}
```
- Load Open Brush.
- Activate the camera tool.

Expected Result: Can see camera preview on each tool, with post-processing effects disabled.
Actual Result: No preview visible.
Platforms reproduced on: Windows 10, Unity editor, Release and Experimental built versions.

![image](https://user-images.githubusercontent.com/9121007/108636821-aaca8600-747f-11eb-9279-56c74090a493.png)

**Comment by mikeskydev** [comment]
This appears to be somehow related to the Sonic Ether plugin. 
Disabling either tilt shift or vignette components separately does not cause this issue, only when both are disabled together. Moving Sonic Ether to the bottom of the components stack achieves the same bug. Deleting the Sonic Ether component prevents the bug from happening.

**Comment by mikeskydev** [comment]
After checking in renderdoc, it appears the image is being generated, but something is up with the alpha value. This is calculated here:

https://github.com/sonicether/SE-Natural-Bloom-Dirty-Lens/blob/bc15de63773354ca99d2a24abe065814e8ba60cf/Assets/SE%20Natural%20Bloom/Shaders/SENaturalBloomAndDirtyLens.shader#L515

Renderdoc with RGBA
![image](https://user-images.githubusercontent.com/9121007/114102008-7b9c9680-98be-11eb-815e-4d93ff16072c.png)

Renderdoc with just RGB
![image](https://user-images.githubusercontent.com/9121007/114102023-86572b80-98be-11eb-8d5c-4a67535ff221.png)

**Comment by mikeskydev** [comment]
Fixing alpha value at 1.0 causes some brushes not to be rendered.

**Comment by andybak** [comment]
Any thoughts on why this breaks in Open Brush but not on the original Tilt Brush? What did we change in this area?

**Comment by mikeskydev** [comment]
These shaders were closed source in TB, and they were been swapped out with open source equivalents for release, but I'm guessing there might be a few differences.

**Comment by andybak** [comment]
Could we switch to the standard Unity Post Processing v2 Package? I don't think it has tilt shift but converting the current code to work with PP wouldn't be hard. Is SE Bloom more performant than the Unity Bloom?

**Comment by mikeskydev** [comment]
Taking a look at this again, armed with the knowledge that the output screenshot actually looks fine! instead I fixed the output alpha to 1.0 on the shader used for the preview window, rather than the capture process, which appears to produce the expected results.

https://github.com/mikeskydev/open-brush/blob/48522725812a78d99afbf5c00fd72cbb5f2b1491/Assets/Shaders/SnapshotCameraFlash.shader#L76

This still doesn't really explain why it is breaking the preview, but I'm thinking it doesn't really matter too much?

**Comment by mikeskydev** [comment]
Just noting that the screenshot manager explicitly ignores alpha when exporting, which is probably why it doesn't encounter this issue. https://github.com/icosa-gallery/open-brush/blob/main/Assets/Scripts/ScreenshotManager.cs#L464-L481
